bin/configs/java-microprofile-rest-client-3.0-jackson.yaml

```diff
@@ -10,3 +10,4 @@ additionalProperties:
   microprofileRestClientVersion: "3.0"
   hideGenerationTimestamp: true
   useReflectionEqualsHashCode: true
+  openApiNullable: false
```

Comment:
This can be removed now, since openApiNullable is being disabled for microprofile. Same comment applies to `bin/configs/java-microprofile-rest-client-3.0.yaml`.